lib/Hyena/Hyena/Hyena.csproj

```diff
@@ -179,9 +179,8 @@
     <Reference Include="glib-sharp, Version=2.12.0.0, Culture=neutral, PublicKeyToken=35e10195dab3c99f">
       <Package>glib-sharp-2.0</Package>
     </Reference>
-    <Reference Include="nunit.framework, Version=2.6.3.0, Culture=neutral, PublicKeyToken=96d09a1eb7f44a77">
-      <Package>nunit</Package>
-      <SpecificVersion>False</SpecificVersion>
+    <Reference Include="nunit.framework">
+      <HintPath>..\..\..\packages\NUnit.3.0.1\lib\net45\nunit.framework.dll</HintPath>
```

Comment:
We might want to temporarily move back to nunit 2.6.4 because Monodevelop doesn't support nunit 3 yet. Is there a feature(s?) in nunit 3 that would be really useful, helpful.